modules/users/tests/server/user.server.model.tests.js

```diff
@@ -153,6 +150,8 @@ describe('User Model Unit Tests:', function () {
   });
 
   after(function (done) {
-    User.remove().exec(done);
+    User.remove().exec(function() {
```

Comment:
When doing `User.remove().exec(done);`, the `after()` would fail? That seems weird..I've never run into that issue and it's not like something crazy is happening. Does it work consistently in other callbacks? What about in the `afterEach()` below?